modules/_node-scripts/configuration/getGlobalSymbols.mjs

```diff
@@ -0,0 +1,21 @@
+import path from 'path';
+
+import getRootDir from '../util/getRootDir.mjs';
+import require from '../util/require.mjs';
+
+/**
+ * @returns
+ * Something like:
+ *
+ * {
+ *   '@clayui/charts': ['__esModule', 'bb', 'default']
+ * }
+ */
+export default async function getGlobalSymbols() {
+	const rootDir = await getRootDir();
+
+	const {symbols} = require(path.join(rootDir, 'node-scripts.config.js'));
```

Comment:
`require`? Shouldn't this be `import()`

Reply:
oh, I just realized you imported a helper called "require". Can we rename that to avoid confusion with cjs?

Reply:
even just requireUtil is fine, my brain will always see require() as cjs

Reply:
I ended up naming it `projectScopeRequire` since it's what it does.